app/src/ui/molecules/sidebar-shop/SidebarShop.js

```diff
@@ -0,0 +1,14 @@
+import CustomLink from "../../atoms/custom-link/CustomLink";
+
+
+const SidebarShop = () => {
+  return (
+    <div className="mt-3 p-4 es-sidebar-shop-bg">
```

Comment:
jak dasz tutaj text-white to nie będziesz musiał dawać tego do każdego dziecka tego diva

Reply:
ok